pkg/common/cert/cert.go

```diff
@@ -0,0 +1,35 @@
+package cert
+
+import (
+	"encoding/pem"
+	"fmt"
+	"os"
+)
+
+func CreateTmpDerFileBasedOnPem(block *pem.Block) (*string, func(), error) {
+	tempDerFile, err := os.CreateTemp("", "tmp-cert-*.der")
```

Comment:
Where do you clear/delete the temp file? In my solution it got auto-deleted thanks to `defer` keyword

Reply:
this function to defer is returned, and it must be defered by function that run "CreateTmpDerFileBasedOnPem". I didn't found any other possibilities to remove duplication of code